<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBarang</class>
 <widget class="QDialog" name="DialogBarang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>#</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Kode barang</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtKode"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbahKode">
       <property name="text">
        <string>Ubah kode...</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nama barang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtNama"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Satuan harga barang (ons, pak, kg, dsb.)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSatuan"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Harga beli (kulak) barang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtHargaBeli"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Harga jual barang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtHargaJual"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Jumlah stok barang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtJumlahStok"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnTambahPerbaruiBarang">
     <property name="text">
      <string>##TAMBAH / PERBARUI BARANG##</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
